scripts/start-deployLoliserver

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# shellcheck source=start-utils
+. "${SCRIPTS:-/}start-utils"
+set -o pipefail
+set -e
+
+latestAsset=$(
+  curl -fsSL https://cdn.ci.loliidc.cn:30011/job/LoliServer-1.16.5/lastSuccessfulBuild/artifact/projects/LoliServer/build/libs | \
```

Comment:
Overall your code looks great. My concern is accessing this URL especially what appears to be an ephemeral port. 

Please try to use the artifact from the latest GitHub action such as https://github.com/Loli-Server/LoliServer/actions/runs/2195635097#artifacts

Reply:
`latestAssetID=$(
  curl -sL https://api.github.com/repos/Loli-Server/LoliServer/actions/workflows/gradle.yml/runs | jq -r '.workflow_runs[0].id?'
)

latestAsset=$(
  curl -fsSL https://api.github.com/repos/Loli-Server/LoliServer/actions/artifacts/${latestAssetID}/zip | \
    jq '.assets[] | select(.name | match(".*.zip"))'
)`

When i try to get the latest artifact id and try to get that `.zip` package, curl throws a `403 Forbidden` on my head. Not sure what iam doing wrong.

`curl -H "Accept: application/vnd.github.v3+json" -o "$SERVER" -fsSL https://api.github.com/repos/Loli-Server/LoliServer/actions/artifacts/${latestAsset}/zip`

Reply:
It looks like you need to grab the artifacts list from the run and use the `archive_download_url` from that:

```
curl --request GET \
  --url https://api.github.com/repos/Loli-Server/LoliServer/actions/runs/2195635097/artifacts
```

Reply:
nm, the variable names were confusing me. I'm getting a 403 also using curl. That API seems to require the caller be authenticated, which is does seem to imply with

> Anyone with read access to the repository can use this endpoint

Reply:
I was about to say that. Since `curl`ing the artifact to see if its there responds with missing scopes.

```
{
  "message": "You must have the actions scope to download artifacts.",
  "documentation_url": "https://docs.github.com/rest/reference/actions#download-an-artifact"
}
```

Reply:
Well, I guess we're stuck with their Jenkins location. It feels like the README section you added needs some kind of warning/disclaimer...but I'm not sure that it should say. If you can't think of anything either, then I'm good with merging this.

Reply:
I added a small disclaimer in [0b3cc9e](https://github.com/itzg/docker-minecraft-server/commit/0b3cc9e8bda6cba3e0fc198797e7ea623584dcbc)